.github/actions/download-models/requirements.txt

```diff
@@ -114,19 +114,19 @@ fsspec==2025.3.2 \
     --hash=sha256:2daf8dc3d1dfa65b6aa37748d112773a7a08416f6c70d96b264c96476ecaf711 \
```

Comment:
colorama is removed here https://github.com/openvinotoolkit/npu_compiler/pull/90/files#diff-eb022f969b849fbde6bbcfc38e0903d373838671162dab2ce42367dbf7267dd4L105